AsyncDisplayKit/ASControlNode.h

```diff
@@ -119,7 +122,12 @@ typedef NS_OPTIONS(NSUInteger, ASControlState) {
   @param event The event which triggered these control actions. May be nil.
  */
 - (void)sendActionsForControlEvents:(ASControlNodeEvent)controlEvents withEvent:(nullable UIEvent *)event;
-
+#if TARGET_OS_TV
```

Comment:
@aaronschubert0 yes, that name sounds much better.  I'm not experienced with tvOS yet, so I can't weigh in, but if that name makes sense to you we should go with it.